apps/react-app/vite.config.ts

```diff
@@ -46,4 +47,8 @@ export default defineConfig({
   server: {
```

Comment:
We need to add this so we can serve assets in dev

```suggestion
  server: {
  port: 3001,
  origin: "http:localhost:3001"
  }
```